run-benchmark.sh

```diff
@@ -857,83 +876,99 @@ runQueries() {
 		    fi
 		 fi
 		 message "Running $BENCH query ${i} at scale $SCALE"
 		 runQuery "query-${i}.sql"
-	      done
-	   fi
-	}
+      done
+   fi
+}
 
-	checkBenchResults() {
-	  local schema=$1
-	  local iter=$2
+checkBenchResults() {
+    local schema=$1
+    local iter=$2
 
-	  local -a results
+    local -a results
 
-	  # TODO: handle SETSTYLE adhoc and sets
+    # TODO: handle SETSTYLE adhoc and sets
 
-	  local i
-	  local j=0
-	  for i in `seq -w $BENCHMIN $BENCHMAX`; do
+    local i
+    local j=0
+    for i in `seq -w $BENCHMIN $BENCHMAX`; do
 	    let j++
+	    local bench="query-${i}.sql"
 	    local -i errCount=$(checkQueryError "${LOGDIR}/query-${i}.out")
 	    #debug checkBenchResults errCount $errCount
 	    if [[ $errCount -eq 0 ]]; then
-	      local time=$(checkQueryTime "${LOGDIR}/query-${i}.out")
-	      if [[ "$time" != "" ]]; then
-		message "$SCHEMA query-${i}.sql took $time milliseconds"
-		results[$j]=$time
-	      else
-		message "$SCHEMA query-${i}.sql no errors and no time"
-		results[$j]="Nan"
-	      fi
-	    else
-	      message "$SCHEMA query-${i}.sql had $errCount errors"
-	      results[$j]="Err"
-	    fi
-	  done
-
-	  # loop over the variable set of results
-	  echo -n "$SCHEMA results"
-	  if [[ "$iter" != "0" ]]; then
-	     echo -n " for run $iter"
-	  fi
-	  echo -n ": "
-	  local -i k=1
-	  while [ $k -le $j ]; do
-	    if (( $k == $j )); then
-	      echo ${results[$k]}
+            local time=$(checkQueryTime "${LOGDIR}/query-${i}.out")
+            if [[ "$time" != "" ]]; then
+		        message "$SCHEMA query-${i}.sql took $time milliseconds"
+		        results[$j]=$time
+		        EXEC_TIME[$bench]=$time
+            else
+		        message "$SCHEMA query-${i}.sql no errors and no time"
+		        results[$j]="Nan"
+		        EXEC_STATUS[$bench]="2"
+            fi
 	    else
-	      echo -ne "${results[$k]}, "
+            message "$SCHEMA query-${i}.sql had $errCount errors"
+            results[$j]="Err"
+            EXEC_STATUS[$bench]="3"
 	    fi
-	    let k++
-	  done
-
-	}
+    done
+
+    # loop over the variable set of results
+    echo -n "$SCHEMA results"
+    if [[ "$iter" != "0" ]]; then
+        echo -n " for run $iter"
+    fi
+    echo -n ": "
+    local -i k=1
+    while [ $k -le $j ]; do
+        if (( $k == $j )); then
+            echo ${results[$k]}
+        else
+            echo -ne "${results[$k]}, "
+        fi
+        let k++
+    done
+
+    # create report for Jenkins
+    for bench in ${!EXEC_START_DATE[@]}; do
+        local date=${EXEC_START_DATE[$bench]}
+        local elapsed=${EXEC_TIME[$bench]}
+        local status="PASS"
+        local errorCode=${EXEC_STATUS[$bench]}
+        if [[ $errorCode != 0 ]]; then
+            status="FAIL"
+            BENCH_PASS="FAIL"
+        fi
+	    echo "$date|$bench|$iter|$status|$errorCode||$elapsed" >> ${LOGBASE}logs/test_run.csv
+    done
+}
 
-	# TOODO: iterate over many results
+	# TODO: iterate over many results
```

Comment:
these spelling errors serve a purpose (TODO is an immediate change, and TOODO is a long term change). please do not modify these comments unless you plan to implement them

Reply:
I intended to restore those and I thought I did but somehow my undo didn't propagate to commits.
I thought the misspelling was deliberate but failed to see how (my interpretation was more like 'do too' which didn't make much sense). IDE is not happy with those but IDE is not the preferable way to do bash programming.